assets/scss/components/_social-links.scss

```diff
@@ -0,0 +1,12 @@
+/* ==========================================================================
+   #SOCIAL LINKS
+   ========================================================================== */
+   
+.c-social-links {
+    @include list-bare();
```

Comment:
If this is something you need to include for all lists, maybe it would be worth setting this as a default styles in `elements/list` as it's used for `ul` ?

```
ul {
 [ ... styles from list-bare mixin ]
}
```